qemu/tests/block_resize.py

```diff
@@ -73,7 +73,7 @@ def compare_block_size(session, block_cmd, block_pattern):
 
     drive_path = ""
     if params.get("os_type") == 'linux':
-        drive_id = params["blk_extra_params_%s" % data_image].split("=")[1]
+        drive_id = params["blk_extra_params_%s" % data_image].split(",")[0].split("=")[1]
```

Comment:
different style for blk_extra_params_stg, you can use function to define it

blk_extra_params_stg = "serial=TARGET_DISK0"
blk_extra_params_stg = "wwn=0x5000123456789abc"